Source/CombatExtended/Compatibility/Race AutoPatchers/RaceUtils.cs

```diff
@@ -11,6 +11,21 @@ namespace CombatExtended.Compatibility
 {
     public static class RaceUtil
     {
+        public static SimpleCurve SharpCurve
+        {
+            get
+            {
+                return new SimpleCurve { new CurvePoint(0.2f, 1f), new CurvePoint(2.0f, 20f) };
+            }
+        }
+
+        public static SimpleCurve BluntCurve
+        {
+            get
+            {
+                return new SimpleCurve { new CurvePoint(0.2f, 2f), new CurvePoint(2.0f, 40f) };
+            }
+        }

```

Comment:
This has so much whitespace it's sorta racist.

```suggestion
        public static SimpleCurve SharpCurve() => new SimpleCurve { new CurvePoint(0.2f, 1f), new CurvePoint(2.0f, 20f);

        public static SimpleCurve BluntCurve() => new SimpleCurve { new CurvePoint(0.2f, 2f), new CurvePoint(2.0f, 40f);
```

(Haven't checked that it actually compiles, I don't remember if that's the exact syntax :D)